partner_multi_relation/readme/USAGE.rst

```diff
@@ -0,0 +1,102 @@
+Relation Types
```

Comment:
https://github.com/OCA/maintainer-tools/blob/master/template/module/readme/USAGE.rst
"it MUST NOT contain reStructuredText sections only body text (paragraphs, lists, tables, etc)"

I'm not sure what the consequences are when you use sections in the usage, description etc. But is says here that you must not do it :S

Reply:
Well, it depends on the paragraph level hierarchy. It would fail in CI if it wasn't correct

Reply:
The only level allows here is `~~~~~~`

Reply:
Thanks!

Reply:
Oh so it must be changed?

Reply:
I'm checking and it seems it's correctly rendered... @sbidoul `-----` titles are tolerated inside this section?

Reply:
`~` is accepted yes, other may work or break anytime.

Reply:
@ddufresne can you please switch to this title style for avoiding problems?